apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -170,6 +178,10 @@ def parse_path(path = params[:filepath], filesystem = params[:fs])
     elsif ::Configuration.files_app_remote_files? && filesystem != 'fs'
       @path = RemoteFile.new(normal_path, filesystem)
       @filesystem = filesystem
+    else
+      @path = PosixFile.new(normal_path)
+      @filesystem = filesystem
+      raise StandardError, "Remote file support is not enabled"
```

Comment:
Forgot to catch this before, but error messages like this should be internationalizeable.